lib/datapacksexpand.js

```diff
@@ -83,7 +83,11 @@ DataPacksExpand.prototype.getNameWithFields = function(nameFields, dataPackData)
         } else if (dataPackData[key] && typeof dataPackData[key] === "string") {
             return dataPackData[key].replace(/[/\\]+/g,' '); // do not allow directory slashes in names
         } else if (dataPackData[key] != null && typeof dataPackData[key] === "object") {
-            return self.getDataPackFolder(null, dataPackData[key].VlocityRecordSObjectType, dataPackData[key]);
+            let sObjDataRecordType = dataPackData[key].VlocityRecordSObjectType;
+            if (jobInfo.upgradeDataPackFields && this.vlocity.datapacksutils.dataPacksExpandedDefinition.StmpaExcludedSObjectsTypes.includes(sObjDataRecordType)) {
```

Comment:
Please use the full same pattern as everything else in dataPacksExpandedDefinition through DataPacksUtils